tests/clickhouse-test

```diff
@@ -869,6 +871,7 @@ def main(args):
     global stop_time
     global exit_code
     global server_logs_level
+    global j2env
```

Comment:
We use it as global only in `configure_testcase_args`, maybe better to pas explicitly?

Reply:
Ok (I did it by analogy with other, clickhouse-test requires major refactoring anyway)